### PR TITLE
Enable Nulls on Contrib.Instrumentation.AWS & Test Projects

### PR DESCRIPTION
**Pull request by FernandoNunes**
Towards #894

## Changes

Enabled Nulls on Contrib.Instrumentation.AWS & Test Projects

### CODE REVIEW

File: test/OpenTelemetry.Contrib.Instrumentation.AWS.Tests/OpenTelemetry.Contrib.Instrumentation.AWS.Tests.csproj [hunk 1]
```diff
@@ -4,6 +4,23 @@
...
+    <DefineConstants>$(DefineConstants)TRACE</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net452|AnyCPU'">
+    <DefineConstants>$(DefineConstants)TRACE</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net452|AnyCPU'">
+    <DefineConstants>$(DefineConstants)TRACE</DefineConstants>
   </PropertyGroup>
```
Comment (Kielek) [inline test/OpenTelemetry.Contrib.Instrumentation.AWS.Tests/OpenTelemetry.Contrib.Instrumentation.AWS.Tests.csproj:24]: Why this section is needed?

**Review by Kielek — COMMENTED** [review]
You still have some compilation errors to resolve before review.

File: test/OpenTelemetry.Contrib.Instrumentation.AWS.Tests/OpenTelemetry.Contrib.Instrumentation.AWS.Tests.csproj (hunk 1 above)
Comment (FernandoNunes) [reply test/OpenTelemetry.Contrib.Instrumentation.AWS.Tests/OpenTelemetry.Contrib.Instrumentation.AWS.Tests.csproj]: This was added by Visual Studio when I set NET452 as a variable for testing nullables on NET452 platform.
Sorry didn't noticed it, removed in next commit.

**Comment by FernandoNunes** [comment]
Hi @Kielek can you aprove this PR workflows to see if it checks in correctly ?

**Comment by FernandoNunes** [comment]
The test that failed was on a project that wasn't even touched, and on my dev machine locally all tests pass.

[xUnit.net 00:00:01.91]     OpenTelemetry.Instrumentation.EventCounters.Tests.EventCountersMetricsTests.EventCounter [FAIL]
[xUnit.net 00:00:01.91]       Assert.Equal() Failure
[xUnit.net 00:00:01.91]       Expected: 1997.0201999999999
[xUnit.net 00:00:01.91]       Actual:   0
[xUnit.net 00:00:01.91]       Stack Trace:
[xUnit.net 00:00:01.91]         D:\a\opentelemetry-dotnet-contrib\opentelemetry-dotnet-contrib\test\OpenTelemetry.Instrumentation.EventCounters.Tests\EventCountersMetricsTests.cs(74,0): at OpenTelemetry.Instrumentation.EventCounters.Tests.EventCountersMetricsTests.EventCounter()
  Failed OpenTelemetry.Instrumentation.EventCounters.Tests.EventCountersMetricsTests.EventCounter [1 s]
  Error Message:
   Assert.Equal() Failure
Expected: 1997.0201999999999
Actual:   0
  Stack Trace:
     at OpenTelemetry.Instrumentation.EventCounters.Tests.EventCountersMetricsTests.EventCounter() in D:\a\opentelemetry-dotnet-contrib\opentelemetry-dotnet-contrib\test\OpenTelemetry.Instrumentation.EventCounters.Tests\EventCountersMetricsTests.cs:line 74

**Review by srprash — APPROVED** [review]

**Review by Kielek — APPROVED** [review]

**Pull request merged**